mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -266,14 +282,23 @@ CLASS net/minecraft/server/MinecraftServer
 	METHOD method_3815 setPvpEnabled (Z)V
 		ARG 1 pvpEnabled
 	METHOD method_3816 isDedicated ()Z
+		COMMENT Checks whether this server is a dedicated server.
+		COMMENT
+		COMMENT <p>A dedicated server refers to a Minecraft server implementation which is detached from a parent Minecraft client process.
+		COMMENT A dedicated Minecraft server only accepts remote connections.
 	METHOD method_3818 getServerMotd ()Ljava/lang/String;
 	METHOD method_3819 getServerIp ()Ljava/lang/String;
 	METHOD method_3820 isLoading ()Z
 	METHOD method_3821 exit ()V
 	METHOD method_3823 setupServer ()Z
+		COMMENT Setups a Minecraft server to be ready for players to connect.
+		COMMENT This method does several things including loading server properties and loading worlds.
+		COMMENT
+		COMMENT @return true if the Minecraft server was successfully setup, false if the server failed to be setup.
 	METHOD method_3826 getServerStartTime ()J
 	METHOD method_3827 getVersion ()Ljava/lang/String;
 	METHOD method_3828 isOnlineMode ()Z
+		COMMENT Checks whether this Minecraft server should require all connected players are using a licensed Minecraft account when connecting to this server.
```

Comment:
I think `Minecraft account` is misleading because it usually refers to the account type used before the account migration. See https://help.minecraft.net/hc/en-us/articles/360034636772-Minecraft-Java-Edition-Account-Types.

Reply:
I could probably say it's referring to a license to Minecraft as the game.

Reply:
Maybe just say requires connecting players to be authenticated by mojang. This is more accurate for why proxies need online modes turned off.

Reply:
Yeah that could work, I would probably omit the bit about proxies doing auth since that is implementation detail.